Made Modals out of the forms

Rusty89 opened the pull request:
I made the forms modals that close when you press escape or submit them

Rusty89 committed:
Made Modals out of the forms

I made the forms modals that close when you press escape or submit them

RialJohnson commented on client-app/src/app/services/services.component.ts:
```diff
@@ -7,7 +10,22 @@ import { Component, OnInit } from '@angular/core';
+  openDialog(): void {
+    const dialogRef = this.dialog.open(ActivityFormComponent, {
+       height: '400px',
+		width: '600px',
```
Haven't looked at spec at all to know what our options are, but if we can get away with not hard  coding the size that'd be best.

Rusty89 replied:
Shoot i thought i deleted that

RialJohnson reviewed (approved):
Thanks for doing the research and implementing this! I would have liked see this pushed first, but I'll probably end up just stealing your code for my PR later tonight. Unless someone else sees it and we can merge it first.

Rusty89 committed:
Edit services.component.ts

Removing height/width values in dialogRef.  They were there to test some things out, forgot to remove them before the commit.  Now deleted

Rusty89 committed:
Update .travis.yml

Rusty89 committed:
Update .travis.yml

Rusty89 committed:
Update .travis.yml

Rusty89 committed:
Update .travis.yml

Rusty89 commented:
Code was put in another branch and merged, this branch is unnecessary now
